scratch/sim_correlator.py

```diff
@@ -96,6 +101,9 @@ def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
         args.digitisers = args.antennas
     if args.digitiser_address is not None and args.sync_time is None:
         parser.error("--sync-time is required when specifying --digitiser-address")
+    if args.input_labels is None:
+        print("No input labels received")
```

Comment:
I see I left this print statement in. Would you like me to remove? Or move elsewhere? I'm not sure if it's important to note.

Reply:
I think take it out.